subprojects/docs/src/samples/userguide/artifacts/maven/build.gradle

```diff
@@ -27,12 +27,6 @@ uploadArchives {
             }
 //END SNIPPET builder
 //START SNIPPET multiple-poms
```

Comment:
This comment would have to move to line 40 to properly generate the snippet for the docs.